experimental/packages/opentelemetry-instrumentation/test/common/autoLoader.test.ts

```diff
@@ -15,16 +15,28 @@
  */
 
 import { Tracer, TracerProvider } from '@opentelemetry/api';
-import { NOOP_METER_PROVIDER } from '@opentelemetry/api-metrics';
 import * as assert from 'assert';
 import * as sinon from 'sinon';
 import { InstrumentationBase, registerInstrumentations } from '../../src';
+import {
+  Meter,
+  MeterOptions,
+  MeterProvider,
+  NOOP_METER
+} from '@opentelemetry/api-metrics';
 
 class DummyTracerProvider implements TracerProvider {
   getTracer(name: string, version?: string): Tracer {
     throw new Error('not implemented');
   }
 }
+
+class DummyMeterProvider implements MeterProvider {
+  getMeter(name: string, version?: string, options?: MeterOptions): Meter {
+    return NOOP_METER;
```

Comment:
Can this method throw instead like the DummyTracerProvider above?

Reply:
Yes that's possible. Did that in [d04eed3](https://github.com/open-telemetry/opentelemetry-js/pull/3198/commits/d04eed3ef85eefd698f693b04af22aa9329e5004), thanks! :slightly_smiling_face:. 

However, `NOOP_METER` is still used in the SDK's `MeterProvider`. I'm still looking into removing that, but the spec states that meter creation needs to return a valid [no-op implementation](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#meter-creation) when shutdown, while `TracerProvider` [does not](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md#tracer-creation) and [does deliver a "normal" `Tracer`... ](https://github.com/open-telemetry/opentelemetry-js/blob/3db10568be4387ef024aa431ca1addef18c2441e/packages/opentelemetry-sdk-trace-base/src/BasicTracerProvider.ts#L93)

Reply:
Cool, thank you for identifying this problem. I think the tracer provider can be tracked in a separate issue. And we can leave the `NOOP_METER` exported.

Reply:
The spec is the same for tracer and we're not honoring that. The spec wording _if possible_ leaves this slightly open to interpretation but we should probably fix that as a bug in the tracer provider.

Reply:
@legendecas Sounds good, thank you :slightly_smiling_face: I created an issue here https://github.com/open-telemetry/opentelemetry-js/issues/3211
@dyladan Ah, true. I missed that this is written in the "shutdown" section of the TracerProvider spec. :slightly_smiling_face: